tests/conftest.py

```diff
@@ -334,9 +336,43 @@ def minimal_dataframe_lookup() -> dict[str, pd.DataFrame]:
     for transformer in other_nan_transformers:
         min_df_dict[transformer] = nan_df
 
+    if library == "polars":
+        for key in min_df_dict:
+            min_df_dict[key] = pl.from_pandas(min_df_dict[key])
+
     return min_df_dict
 
 
+@pytest.fixture
+def get_assert_frame_equal(request):
```

Comment:
if we pass the frame to this function it can lookup the type to get the appropriate library so we don't have to parameterise it?

Reply:
updated this https://github.com/lvgig/tubular/pull/328/commits/fd33732623e0650fc284db5935468a547ff753ca

Reply:
think this is better but the call to get_assert_frame_equal(df)(df, original_df) is a bit clunky.  Might be better to refactor such that the function runs the check rather than just retuning a function so the call is just get_assert_frame_equal(original_df, df).  Could potentially just wrap the current get_assert_frame_equal_fucntion to achieve this.

If doing so it would probably make sense to name the function to just assert_frame_equal but for the conflict with the pd/pl method names. Maybe that's not really an issue to worry about though?

If it is then maybe: assert_frame_equal_dispatch? assert_pl_or_pd_frames_equal? assert_frameT_equal?

Reply:
updated here, went with assert_frame_equal_dispatch https://github.com/lvgig/tubular/pull/328/commits/fd39b7dced8f724dd5b35d410a595de21635f01f